db-async-common/src/main/java/com/github/jasync/sql/db/Configuration.kt

```diff
@@ -53,7 +53,8 @@ data class Configuration @JvmOverloads constructor(
     val applicationName: String? = null,
     val interceptors: List<Supplier<QueryInterceptor>> = emptyList(),
     val eventLoopGroup: EventLoopGroup = NettyUtils.DefaultEventLoopGroup,
-    val executionContext: Executor = ExecutorServiceUtils.CommonPool
+    val executionContext: Executor = ExecutorServiceUtils.CommonPool,
+    val currentSchema: String? = null,
```

Comment:
maybe just call the parameter "schema"?

Reply:
It's `currentSchema` on the URL so I think it makes sense to keep it as that.